backend/api/chat_orchestrator.py

```
@@ -41,6 +74,411 @@ def __init__(self, scenario_data: Dict[str, Any]):
+            
+            intro += f"\n*You have {self._get_turns_remaining()} turns to achieve the objective.*"
+            
+            # Add context from previous scenes if available
+            if previous_summaries:
+                intro += "\n\n**Previous Context:**\n"
+                for summary in previous_summaries[-2:]:  # Last 2 scenes
+                    try:
+                        summary_data = json.loads(summary.summary_text)
+                        scene_title = summary_data.get('scene_data', {}).get('title', 'Previous Scene')
+                        intro += f"• {scene_title}: Key insights and progress\n"
+                    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)